lib/inquisition/outputter.rb

```diff
@@ -40,6 +41,7 @@ def setup_default
       def find_outputter(outputter_to_use)
         return Outputter::Progress if %w[p progress].include?(outputter_to_use)
         return Outputter::HTML if %w[h html].include?(outputter_to_use)
+        return Outputter::XLSX if %w[h xlsx].include?(outputter_to_use)
```

Comment:
`h` it's a wrong short key and already used by `HTML` outputter. You need to use `x` instead.
```suggestion
        return Outputter::XLSX if %w[x xlsx].include?(outputter_to_use)
```